src/schemas/Program/index.js

```diff
@@ -292,11 +292,6 @@ const resolveHTTPProgram = async (programShortName) => {
 	return response ? formatHttpProgram(response) : null;
 };
 
-const resolveDataCenterList = async (egoToken) => {
-	const response = await programService.listDataCenters(egoToken);
-	return response || null;
-};
-
```

Comment:
This function is not reused anywhere else, so it is removed and lines 296-297 are added to line 381-382.